<launch>
    <!-- motordriver -->
    <node pkg="horiokart2021_sensors" type="serial_motordriver_node" name="serial_motordriver_node" output="screen">
        <param name="device_name" type="string" value="/dev/ttyHoriokart-motordriver" />
        <param name="twist_frame" type="string" value="/cmd_vel" />
        <param name="base_frame" type="string" value="base_footprint" />

        <param name="r_dir" type="bool" value="true" />
        <param name="l_dir" type="bool" value="true" />

        <param name="csv" type="bool" value="true" />
    </node>
    
</launch>
